pynestml/codegeneration/resources_nest/point_neuron/common/SynapseHeader.h.jinja2

```diff
@@ -1344,12 +1337,13 @@ inline void
 {%- endfor %}
 
 #ifdef DEBUG
-    //std::cout << "{{synapseName}}::trigger_update_weight(): \tupdating from " << t_lastspike_ << " to " << t_trig + dendritic_delay << std::endl;
+  std::cout << "[synapse " << this << "] {{ synapseName }}::trigger_update_weight(): \tupdating from " << t_lastspike_ << " to " << t_trig + dendritic_delay << std::endl;
 #endif
 
- /* update_internal_state_(t_lastspike_,
-                         t_trig - t_lastspike_,
-                         cp);*/
+  // N.B. call to update_internal state is not needed; this is already taken care of by ``process_{{vt_port}}_spikes_()``
+  // update_internal_state_(t_lastspike_,
+  //                        t_trig - t_lastspike_,
+  //                        cp);
```

Comment:
Do you want to remove this then?

Reply:
I don't really mind very much, but would slightly prefer to keep these lines, to make explicit what would otherwise happen; hopefully the comment explains it sufficiently.